tests/kernels/test_cutlass_moe.py

```diff
@@ -242,3 +254,117 @@ def test_cutlass_moe_cuda_graph(
                                    cutlass_output,
                                    atol=9e-2,
                                    rtol=1e-2)
+
+
+@pytest.mark.parametrize("m", [2, 64, 224])
+@pytest.mark.parametrize("n", [1024, 3072])
+@pytest.mark.parametrize("k", [1024, 1536])
```

Comment:
I think we should factor this out into a common MNK factors and use it across the various tests in this file, like we do in test_cutlass.py

https://github.com/vllm-project/vllm/blob/1cab43c2d2172459987652a7fbf922c6611a71fa/tests/kernels/test_cutlass.py#L18-L38